proto/desmos/posts/v2/models.proto

```diff
@@ -200,25 +200,18 @@ message Attachment {
     (gogoproto.moretags) = "yaml:\"subspace_id\""
   ];
 
-  // Id of the subspace section inside which the post to which this attachment
-  // should be connected is
-  uint32 section_id = 2 [
-    (gogoproto.customname) = "SectionID",
-    (gogoproto.moretags) = "yaml:\"section_id\""
-  ];
```

Comment:
This has been removed as it was useless (never used anyway since we already have the post id)